es-design-system/package-lock.json

```diff
@@ -2682,18 +2679,18 @@
             }
         },
         "node_modules/@energysage/es-bs-base": {
-            "version": "2.0.0-alpha.0",
-            "resolved": "https://registry.npmjs.org/@energysage/es-bs-base/-/es-bs-base-2.0.0-alpha.0.tgz",
-            "integrity": "sha512-aijqFOPrPgf3iNp7ccqJHDGDgDkiyHduyQV7fxheQU5A+73rSMQ4JXG/z81pXAX7qTabI1A90rtmyJd8GXEIOw==",
+            "version": "2.0.0-alpha.1",
+            "resolved": "https://energysage-659694780082.d.codeartifact.us-east-1.amazonaws.com/npm/cloud-engineering/@energysage/es-bs-base/-/es-bs-base-2.0.0-alpha.1.tgz",
```

Comment:
It looks like some of these dependencies are tied to codeartifact still

Reply:
Not sure why it keeps doing that